applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -267,3 +296,127 @@ int ble_core_init(ble_core_ready_t ready_callback)
 
 	return 0;
 }
+
+static void shell_help_print(const struct shell *shell)
+{
+	shell_print(shell, "Possible TX power values:");
+	shell_print(shell, "\tzero: 0dBm");
```

Comment:
What about +3, +10, and +20?

Reply:
To keep the complexity down I have not included ble_hci_vsc_radio_high_pwr_mode_set() and nRF21540 support for the shell commands. We can discuss if we want to include this.

Added CONFIG_NRF_21540_ACTIVE check to both shell commands.

Reply:
We discussed this, and decided to not implement support for ble_hci_vsc_radio_high_pwr_mode_set() and the nRF21540 as for now